src/mcp/shared/context.py

```diff
@@ -17,4 +21,5 @@ class RequestContext(Generic[SessionT, LifespanContextT, RequestT]):
     meta: RequestParams.Meta | None
     session: SessionT
     lifespan_context: LifespanContextT
+    experimental: Any = field(default=None)  # Set to Experimental instance by Server
```

Comment:
this looks a little fishy, should it really be `Any` or `Experimental | None` or something?